pkg/ccl/backupccl/backupdest/backup_destination.go

```diff
@@ -143,21 +142,6 @@ func ResolveDest(
 		return ResolvedDestination{}, err
 	}
 
-	// At this point, the plannedBackupDefaultURI is the full path for the backup. For BACKUP
```

Comment:
i think chosenSuffix is always non empty? so i think you can debranch the conditional on 127

Reply:
Oh yeah, it looks like backup planning always populates `Subdir`